src/ripple/app/ledger/impl/SkipListAcquire.cpp

```diff
@@ -179,23 +178,19 @@ SkipListAcquire::processData(
     failed_ = true;
     JLOG(journal_.error()) << "failed to retrieve Skip list from verified data "
                            << hash_;
-    notifyTasks(sl);
+    notify(sl);
 }
 
 void
-SkipListAcquire::addTask(std::shared_ptr<LedgerReplayTask>& task)
+SkipListAcquire::addDataCallback(OnSkipListDataCB&& cb)
 {
     ScopedLockType sl(mtx_);
-    tasks_.emplace_back(task);
-    if (failed_)
+    dataReadyCallbacks_.emplace_back(cb);
```

Comment:
If you want to move from `cb`, change this to `dataReadyCallbacks_.emplace_back(std::move(cb));`

Reply:
Thanks, changed.